README.md

```diff
@@ -1,4 +1,10 @@
 DrawBot
 =======
 
-DrawBot is a powerful, free application for MacOSX that invites you to write simple Python scripts to generate two-dimensional graphics. The builtin graphics primitives support rectangles, ovals, (bezier) paths, polygons, text objects and transparency.
\ No newline at end of file
+DrawBot is a powerful, free application for MacOSX that invites you to write simple Python scripts to generate two-dimensional graphics. The builtin graphics primitives support rectangles, ovals, (bezier) paths, polygons, text objects and transparency.
+
+
+This is a private fork just for the purpose of making a 10.6 executable. [Download from my Dropbox.](https://goo.gl/22OHVh)
```

Comment:
The part that says "this is a private fork," as making a PR against mainline is not private fork.